packages/vuepress-theme-odyssey/styles/base/_fonts.scss

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020-Present Okta, Inc.
```

Comment:
Please remove this file!

Reply:
Weird, not sure how this got here. Removed.